paper-radio-button.html

```diff
@@ -65,27 +68,32 @@
       }
 
       #radioContainer {
-        display: inline-block;
+        @apply(--layout-inline);
+        @apply(--layout-center-center);
         position: relative;
-        width: 16px;
-        height: 16px;
+        width: var(--calculated-paper-radio-button-size);
+        height: var(--calculated-paper-radio-button-size);
         vertical-align: middle;
       }
 
       #ink {
         position: absolute;
-        top: -16px;
-        left: -16px;
-        width: 48px;
-        height: 48px;
+        top: 50%;
+        left: 50%;
+        width: calc(2.66 * var(--calculated-paper-radio-button-size));
+        height: calc(2.66 * var(--calculated-paper-radio-button-size));
```

Comment:
Shouldn't this be 3? (previous ink width `48px` / previous container width `16px`)